src/st/no2_psp/st_init_no2.c

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "../no2/no2.h"
+
+#include "../pfn_entity_update.h"
+
+extern LayoutEntity* D_8D2E1B4;
+extern LayoutEntity* D_8D2E288;
+
+extern Overlay OVL_EXPORT(Overlay);
+
+s32 E_ID(ID_11);
```

Comment:
I opted for E_UNK_11 (for example) instead of E_ID_11.  It better communicates that we don't know what it is for yet.